indico/modules/events/contributions/templates/management/_contribution_list.html

```diff
@@ -115,18 +115,26 @@
                                 </a>
                             </td>
                             <td class="i-table" style="text-align: right;">
+                                <a href="#" class="icon-list"
+                                   title="{% trans %}Manage subcontributions{% endtrans %}"
+                                   data-href="{{ url_for('.manage_subcontributions', contrib) }}"
+                                   data-title="{% trans %}List of the subcontributions{% endtrans %}"
+                                   data-ajax-dialog>
+                                </a>
                                 <a href="#" class="icon-edit"
-                                   data-ajax-dialog
                                    data-href="{{ url_for('.manage_update_contrib', contrib) }}"
                                    data-title="{% trans title=contrib.title %}Edit contribution '{{ title }}'{% endtrans %}"
-                                   data-update="#contribution-list">
+                                   data-update="#contribution-list"
+                                   data-ajax-dialog>
                                 </a>
                                 <a href="#" class="icon-attachment"
-                                    data-attachment-editor
+                                    title="{% trans %}Manage contribution attachments{% endtrans %}"
                                     data-title="{% trans title=contrib.title %}Manage materials for '{{ title }}'{% endtrans %}"
-                                    data-locator="{{ contrib.locator|tojson|forceescape }}">
+                                    data-locator="{{ contrib.locator|tojson|forceescape }}"
```

Comment:
not sure why my initial comment didn't end up being saved.. anyway, it was something like this

> no fixed rule for it, use whatever is more readable. i usually go for spaces around the pipe. there are some cases where it's more readable without though